www/pages/_document.tsx

```diff
@@ -3,7 +3,7 @@ import Document, { Html, Head, Main, NextScript } from 'next/document'
 export default class MyDocument extends Document {
   render() {
     return (
-      <Html lang="en" className="dark">
+      <Html lang="en" className="dark overflow-x-hidden">
```

Comment:
No because it's linked to FlyOut width issue.
![image](https://user-images.githubusercontent.com/69633530/112822230-f7f5d500-9087-11eb-8668-c6e1014a685e.png)